### DISCUSSION TITLE
fcitx5 服务首次运行时是否可以自动检查和配置运行环境？

### DISCUSSION THREAD

**Discussion by cheese** [fcitx/fcitx5#478] [Ideas]
1. 自动注入所需环境变量，提示用户下次登录时生效
2. 自动检查异常的环境变量，提示用户是否需要将默认输入法改为 Fcitx，并自动帮用户配置
3. 检查用户运行环境，提示用户安装必要的 im module

**Comment by wengxt** [comment]
合理的环境变量 setup 有很多组合，而且我比较不喜欢这样的侵入式配置。不会考虑实现。
